src/workerd/api/wpt/url-test.js

```diff
@@ -0,0 +1,17 @@
+import * as harness from 'harness';
+
+export const urlConstructor = {
+  async test() {
+    harness.prepare();
+    await import('url-constructor.any.js');
+    harness.validate();
```

Comment:
Side note: Our workerd tests could benefit from having a `before() { ... }` and `after() { ... }` mechanism.

Reply:
I suppose we could have a pattern like...

```
import * as harness from 'harness';

export const urlConstructor = {
  test: harness.test('url-constructor.any.js');
};
```

Where the `harness.test(...)` handles any of the necessary boilerplate for the most typical cases?

Reply:
Yeah sounds like a good idea